Description: Correct hyphens used as minus signs
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: bc/doc/bc.1
===================================================================
--- bc.orig/doc/bc.1	2013-04-21 16:25:33.292838937 -0400
+++ bc/doc/bc.1	2013-04-21 16:25:33.288838837 -0400
@@ -543,7 +543,7 @@
 variable \fBpi\fR.
 .RS
 \f(CW
-pi=$(echo "scale=10; 4*a(1)" | bc -l)
+pi=$(echo "scale=10; 4*a(1)" | bc \-l)
 \fR
 .RE
 .PP
@@ -565,7 +565,7 @@
   /* Check the sign of x. */
   if (x<0) {
     m = 1
-    x = -x
+    x = \-x
   } 
 
   /* Precondition x. */
@@ -615,7 +615,7 @@
   "current balance = "; bal
   "transaction? "; trans = read()
   if (trans == 0) break;
-  bal -= trans
+  bal \-= trans
   bal /= 1
 }
 quit
@@ -641,7 +641,7 @@
 to \fBbc\fR.  It also allows for a history of previous lines typed.
 When this option is selected, \fBbc\fR has one more special variable.
 This special variable, \fBhistory\fR is the number of lines of history
-retained.  For \fBreadline\fR, a value of -1 means that an unlimited
+retained.  For \fBreadline\fR, a value of \-1 means that an unlimited
 number of history lines are retained.  Setting the value of
 \fBhistory\fR to a positive number restricts the number of history
 lines to the number given.  The value of 0 disables the history
@@ -658,7 +658,7 @@
 It is not implemented in the traditional way using
 .I dc(1).
 This version is a single process which parses and runs a byte code
-translation of the program.  There is an "undocumented" option (-c)
+translation of the program.  There is an "undocumented" option (\-c)
 that causes the program to output the byte code to
 the standard output instead of running it.  It was mainly used for
 debugging the parser and preparing the math library.
@@ -710,13 +710,13 @@
 .IP "function format"
 POSIX \fBbc\fR requires the opening brace on the same line as the 
 \fBdefine\fR key word and the \fBauto\fR statement on the next line.
-.IP "=+, =-, =*, =/, =%, =^"
+.IP "=+, =\-, =*, =/, =%, =^"
 POSIX \fBbc\fR does not require these "old style" assignment operators to
 be defined.  This version may allow these "old style" assignments.  Use
 the limits statement to see if the installed version supports them.  If
 it does support the "old style" assignment operators, the statement
-"a =- 1" will decrement \fBa\fR by 1 instead of setting \fBa\fR to the
-value -1.
+"a =\- 1" will decrement \fBa\fR by 1 instead of setting \fBa\fR to the
+value \-1.
 .IP "spaces in numbers"
 Other implementations of \fBbc\fR allow spaces in numbers.  For example,
 "x=1 3" would assign the value 13 to the variable x.  The same statement
Index: bc/doc/dc.1
===================================================================
--- bc.orig/doc/dc.1	2013-04-21 16:25:33.292838937 -0400
+++ bc/doc/dc.1	2013-04-21 16:25:42.769074416 -0400
@@ -26,9 +26,9 @@
 .SH NAME
 dc \- an arbitrary precision calculator
 .SH SYNOPSIS
-dc [-V] [--version] [-h] [--help]
-   [-e scriptexpression] [--expression=scriptexpression]
-   [-f scriptfile] [--file=scriptfile]
+dc [\-V] [\-\-version] [\-h] [\-\-help]
+   [\-e scriptexpression] [\-\-expression=scriptexpression]
+   [\-f scriptfile] [\-\-file=scriptfile]
    [file ...]
 .SH DESCRIPTION
 .PP
